integrations/rabbitmq/documentation.yaml

```diff
@@ -0,0 +1,95 @@
+exporter_type: included
+app_name_short: RabbitMQ
+app_name: {{app_name_short}}
+app_site_name: RabbitMQ
+app_site_url: https://rabbitmq.com/
+exporter_name: the RabbitMQ exporter
+exporter_pkg_name: rabbitmq
+exporter_repo_url: https://rabbitmq.com/prometheus.html#overview-prometheus
+minimum_exporter_version: v3.8.0
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting metrics on the expected
+  endpoints, set up port-forwarding with the following command:
+
+  To determine if {{exporter_name}} is exposing metrics, run the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl -n {{namespace_name}} port-forward {{pod_name}} 15692
+  </pre>
+
+  Access the endpoint `localhost:15692/metrics` by using the browser or curl in another terminal session
+  to verify that the metrics are being exposed by the exporter for scraping.
+dashboard_available: true
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: rabbitmq
+    labels:
+      app.kubernetes.io/name: rabbitmq
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: prometheus
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/component: rabbitmq
+additional_podmonitoring_info: |
+  Ensure that the values of the port and matchLabels match that of the {{app_name_short}} pods you want to monitor.
```

Comment:
\`port\` and \`matchLabels\` fields match those of

(literal "port" and "matchLabels", add "fields", and there's more than one, so "match those", not "match that".